user-management/create_users.py

```diff
@@ -0,0 +1,42 @@
+import csv
+import fire
+import json
+import os
+import sys
+
+current_dir = os.path.dirname(os.path.realpath(__file__))
+parent_dir = os.path.dirname(current_dir)
+sys.path.append(parent_dir)
```

Comment:
these are needed to access toolbox.* ya?

Reply:
iya betul No, atau mungkin ada saran lain?

Reply:
aku ada pikiran untuk di-duplikat aja, dan folder ini bisa standalone gt 🤔 
jadi users memang clone 1 repo, tapi kalau hanya butuh user management, ya bisa copy 1 folder aja gt

tapi kayanya masih cukup simple dan masih ok diginiin dulu

mungkin di-note di readme dan code, kalau create_users.py ini depend ke toolbox jg

Reply:
udpated https://github.com/datasaur-ai/api-client-example/pull/22/commits/3fc2b93cc613c235eb44af7acedeaa869fd0815f